src/main/java/duke/command/ChangeStatusCommand.java

```diff
@@ -0,0 +1,55 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.util.TaskList;
+import duke.util.Storage;
+import duke.util.Ui;
+
+/**
+ * Represents a command to change the status of task in the task list.
+ */
+public class ChangeStatusCommand extends Command {
+    int taskIndex;
+    boolean isDone;
```

Comment:
Good naming conventions were used however I noticed there are missing access modifiers here as well as other places in your other classes. Perhaps a thought to consider would be if no access modifier is stated, what would it default to, and whether that is desirable?